src/main/model/ProxyCommand.java

```diff
@@ -0,0 +1,37 @@
+package main.model;
+
+public class ProxyCommand {
+    private String url;
+    private String user;
+    private String server;
```

Comment:
You'd probably want to change this to something like ``endpoint`` since we don't usually have a specific server as the endpoint. :)